share/picongpu/examples/LandauDamping/include/picongpu/param/density.param

```diff
@@ -0,0 +1,77 @@
+/* Copyright 2013-2018 Axel Huebl, Jakob Ameres
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "picongpu/particles/densityProfiles/profiles.def"
+
+
+namespace picongpu
+{
+namespace SI
+{
+    /** Base density in particles per m^3 in the density profiles.
+     *
+     * This is often taken as reference maximum density in normalized profiles.
+     * Individual particle species can define a `densityRatio` flag relative
+     * to this value.
+     *
+     * unit: ELEMENTS/m^3
+     */
+    constexpr float_64 BASE_DENSITY_SI = 1.;
+}
+
+namespace densityProfiles
+{
+    struct CosinePertubationFunctor
+    {
+        /** This formula uses SI quantities only.
+         *  The profile will be multiplied by BASE_DENSITY_SI.
+         *
+         * @param position_SI total offset including all slides [meter]
+         * @param cellSize_SI cell sizes [meter]
+         *
+         * @return float_X density [normalized to 1.0]
+         */
+        HDINLINE float_X
+        operator()(
+            const floatD_64& position_SI,
+            const float3_64& cellSize_SI
+        )
+        {
+            /* calculate cells -> SI [m] */
+            float_64 const x_SI = position_SI.x();
+
+            /* specify your E-Field in V/m and convert to PIConGPU units */
+            float_X const arg = precisionCast< float_X >( x_SI * k_0 );
+            float_X const eps = 0.05;
+
+            // range: [0.0:2.0]
+            float_X const s = 1. + eps * math::cos( arg );
+
+            /* all parts of the function MUST be > 0 */
+            s *= float_64(s >= 0.0);
```

Comment:
please use `float_X(s >= 0.0)` this is only a cast from bool into `float_X` and `s` is also a `float_X`

Reply:
ups, need to change this in the default example as well